### PR TITLE
style: 🎨 修改脚手架命令行列表

### PR DESCRIPTION
**Pull request by xun082**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/xun082/create-neat/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[√ ] Other... Please describe:
```

## What is the current behavior?

- Closes #
- Related to #

## What is the new behavior?

修改脚手架命令行列表

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/core/src/types/index.ts
Comment (IsDyh01) [inline packages/core/src/types/index.ts:1]: 把插件分开之后后边要改的就太多了

File: packages/core/src/types/index.ts [hunk 1]
```diff
@@ -31,3 +31,11 @@ export interface PackageJsonType {
 }
 
 export type buildToolType = "webpack" | "vite" | "rollup";
+
```
Comment (uaenaTzx) [inline packages/core/src/types/index.ts:34]: 昨天说的就是这一块，没必要的

Comment (IsDyh01) [reply packages/core/src/types/index.ts:34]: 不是说要按照m说的写吗

Comment (uaenaTzx) [reply packages/core/src/types/index.ts:34]: 选项按照 m 的写，类型不用写那么细

Comment (IsDyh01) [reply packages/core/src/types/index.ts:34]: 这个是有什么影响吗

Comment (uaenaTzx) [reply packages/core/src/types/index.ts:34]: 逻辑上没影响，但过度的 ts 影响代码风格了

Comment (IsDyh01) [reply packages/core/src/types/index.ts:34]: 那现在是把这些类型都给删掉吗

Comment (uaenaTzx) [reply packages/core/src/types/index.ts:34]: 对于插件功能细分的类型可以删了

**Review by uaenaTzx — APPROVED** [review]

**Pull request merged**